e2e/test/utils/app.helper.ts

```diff
@@ -44,6 +44,17 @@ class AppHelper {
     await driver.saveScreenshot(`${screenshotsFolder}/${fileName}.png`);
   }
 
+  /**
+   * Remove the popover if it exists
+   */
+  async removePopover() {
+    const popoverId = `//*[@resource-id="closePopover"]`;
```

Comment:
Was this meant to be called `closePopoverId`?

Reply:
True! Fixed.